bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/handler/OpenSprinklerHttpBridgeHandler.java

```diff
@@ -92,6 +92,14 @@ public void refreshStations() {
         }
     }
 
+    public void delayedRefresh() {
+        try {
+            Thread.sleep(2000);
```

Comment:
I'm a bit shocked to see this pattern from you... This will block the framework's thread. Why not scheduling a task?

Reply:
I'm self taught with no formal training and most experience before java was with microprocessors that don't have threads. Thanks for pointing this out, I have done some reading up on it and have a better grasp on it.